lib/clearance/configuration.rb

```diff
@@ -93,6 +98,7 @@ def initialize
       @cookie_name = "remember_token"
       @httponly = false
       @mailer_sender = 'reply@example.com'
+      @password_reset_time_limit = 15.minutes
```

Comment:
you could make `15` the env var and then do `ENV["PASSWORD_RESET_TIME_IN_MINUTES"].to_i.minutes` 

I am not 100% sure it makes sense here, but my general rule in a regular rails app is that time-related variables should be `ENV` vars in case they need to be changed on the fly. For example, on my current project, we have a `time_to_refresh` var that is 15 minutes on staging and prod for performance reasons, but sometimes we want to move it down to 1 minute of we are testing something on staging.

Your call if you want to go that route here, though!